v4/source/jobs.yaml

```diff
@@ -0,0 +1,171 @@
+  '/jobs':
+    get:
+      tags:
+        - jobs
+      summary: Get the jobs.
+      description: |
+        Get a page of jobs. Use the query parameters to modify the behaviour of this endpoint.
+        __Minimum server version: 4.1__
+        ##### Permissions
+        Must have `manage_system` permission.
```

Comment:
Maybe we should add new permissions for jobs since we'll be breaking it all out for advanced permissions anyways